multimodal/tests/unittests/test_predictor.py

```diff
@@ -100,7 +100,7 @@ def verify_realtime_inference(predictor, df, verify_embedding=True):
             "hateful_memes",
             ["timm_image", "t_few", "clip", "fusion_mlp"],
             "t5-small",
-            "swin_tiny_patch4_window7_224",
```

Comment:
did we change the image backbone from swin transformers to mobile net? If so, will it impact ongoing CV benchmarking?

Reply:
No, only for accelerating the speed of the test-case.